pkg/controller/worker/machines.go

```diff
@@ -268,3 +273,32 @@ func (w *workerDelegate) generateMachineConfig(ctx context.Context) error {
 
 	return nil
 }
+
+func (w *workerDelegate) enableAcceleratedNetworking(machineType, machineImageName, machineImageVersion string) bool {
```

Comment:
Can you add a unit test case that uses this accelerated networking?

Reply:
Yep. In general I wanted to add more test for the worker creation.